src/main/osd/osd_elements.c

```diff
@@ -640,20 +640,19 @@ static void osdElementGpsFlightDistance(osdElementParms_t *element)
 static void osdElementGpsHomeDirection(osdElementParms_t *element)
 {
     if (STATE(GPS_FIX) && STATE(GPS_FIX_HOME)) {
-        if (GPS_distanceToHome > 0) {
+        if (GPS_distanceToHome > 5) {
```

Comment:
Maybe 1 is too short, but 6 to me is too far. I think is better to define it like a constant and maybe a value of 2 or 3 is enough. I don't know what others will think.

Reply:
Yeah that's debatable. Given that GPS accuracy at beast is around 2m, I guess it could be lowered.
In iNav, the arrow-to-home turns into the home flag when distance is lower than min_rth_distance, so it also helps to know if the aircraft will RTH or will land, but BF does not have that.

Reply:
Yes, we have something similar, but the RTH of Betaflight is very basic, by default the min rth is 100m :P